providers/aws/services/vpc/vpc_endpoint_services_allowed_principals_trust_boundaries/vpc_endpoint_services_allowed_principals_trust_boundaries.py

```diff
@@ -0,0 +1,39 @@
+from config.config import get_config_var
+from lib.check.models import Check, Check_Report
+from providers.aws.services.vpc.vpc_client import vpc_client
+
+
+class vpc_endpoint_services_allowed_principals_trust_boundaries(Check):
+    def execute(self):
+        findings = []
+        # Get trusted account_ids from config.yaml
+        trusted_account_ids = get_config_var("trusted_account_ids")
```

Comment:
Same as above, we have to check errors here.

Reply:
Done